packages/nimble-components/src/table/template.ts

```diff
@@ -31,7 +31,8 @@ export const template = html<Table>`
         aria-multiselectable="${x => x.ariaMultiSelectable}"
         ${children({ property: 'childItems', filter: elements() })}
     >
-        <div class="table-container" style="
+        <div class="${x => (x.documentShiftKeyDown ? 'disable-select table-container' : 'table-container')}"
```

Comment:
Stealing @rajsite's comment from #1218 since it looks like this PR will likely go in first.

>nit: can split out the constants:

```suggestion
        <div class="table-container ${x => (x.documentShiftKeyDown ? 'disable-select' : '')}"
```

Reply:
Updated